test/WebSites/RequestServicesWebSite/Startup.cs

```diff
@@ -14,6 +16,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddMvc();
 
             services.AddScoped<RequestIdService>();
+            services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
```

Comment:
Is it the plan to remove `IHttpContextAccessor` entirely? Or will the Hosting repo (for example) register it for customer use?